docs/api/release-notes/3.4.0.md

```diff
@@ -0,0 +1,53 @@
+---
+title: PMM 3.4.0 API release notes
+slug: release-notes-3-4-0
+categorySlug: release-notes
+hidden: 0
+---
+
+Here are the key API changes in the PMM 3.4.0 API:
+
+### RDS Instance ID support
+The `/v1/management/RDS/Add` endpoint now supports an optional `instance_id` parameter when adding RDS instances. This enhancement addresses API inconsistencies where PMM would modify node addresses in responses, breaking automation tools.
+
+You can now record the actual AWS RDS Instance ID in PMM, making it easier to match monitored instances with their corresponding resources in the AWS console.
+
+### Affected endpoints
+
+- POST /v1/management/RDS/Add - Now accepts optional `instance_id` parameter
+- GET /v1/management/services - Returns `instance_id` values for RDS nodes
+
+
+### Example request
+```json
+{
+  "node_name": "production_rds",
+  "address": "mydb.cluster.us-east-1.rds.amazonaws.com",
+  "instance_id": "production-mysql-01",
```

Comment:
```suggestion
  "service_name": "production-mysql-01",
  "instance_id": "production-mysql-01",
```
...to make it match the screenshot below, where the "service_name" is provided by the user.